packages/env/lib/config/test/config.js

```diff
@@ -242,7 +251,11 @@ describe( 'readConfig', () => {
 			readFile.mockImplementation( () =>
 				Promise.resolve(
 					JSON.stringify( {
-						plugins: [ './relative', '../parent', '~/home' ],
+						plugins: [
+							'./relative',
+							'../parent',
+							`${ os.homedir() }/home`,
```

Comment:
I use `os.homeDir()` because tilde (`~`) is not recognized as a home directory path in Windows.